### PR TITLE
RDAPI-24211 - Addition of text to Important Changes section of APIM May21 Release Notes

### PR DESCRIPTION
**Pull request by blynch-axway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Adding in text for the Important changes part of the release notes. Some text has already been added separately, this should cover the rest of what is needed.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

https://deploy-preview-1890--axway-open-docs.netlify.app/docs/apim_relnotes/20210530_apimgr_relnotes/

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [ x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ x] You have signed the [Axway CLA](https://cla.axway.com/)
* [ x] You have verified the technical accuracy of your change
* [ x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by blynch-axway — COMMENTED** [review]
Looks fine Andrea - thanks

File: content/en/docs/apim_relnotes/20210530_apimgr_relnotes.md [hunk 1]
```diff
@@ -75,6 +76,31 @@ Host name verification is not enabled for upgrading customers, who use Embedded
...
+
+<!-- RDAPI-23648 -->
+### Permissions on apigateway folder
+
+The default permissions on the `apigateway` folder have been restricted down to a Linux 750 level (read/write/execute for owner, read/execute for group, no permissions for world), to provide additional protection for sensitive content, including files and sub-folders.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210530_apimgr_relnotes.md:92]: @blynch-axway Great work! Just one comment: (The default permissions on the `apigateway` folder) - seems too generic to me. Is just the name of the folder enough, will customers know where it is? Or would be better to add the complete path?

Comment (blynch-axway) [reply content/en/docs/apim_relnotes/20210530_apimgr_relnotes.md:92]: Yes - good point, let me ask Pedro who worked on this to provide.... thanks

Comment (blynch-axway) [reply content/en/docs/apim_relnotes/20210530_apimgr_relnotes.md:92]: Hi @andreamussap - was just on to Pedro and I wasnt aware but there is a separate docs commit in which he and Cian agreed to add in a few lines about this item into the Release notes. See https://github.com/Axway/axway-open-docs/pull/1883/files
Might be best to merge the two texts if possible ? But the path to show in any event is just INSTALL_DIR/apigateway.  Thanks

**Review by andreamussap — APPROVED** [review]

**Pull request merged**